src/homeWork/module7/task1/DisplayTime.java

```diff
@@ -0,0 +1,24 @@
+package homeWork.module7.task1;
+
+/**
+ * Created by : Volodymyr_Silitskyi
+ * Created at : 6/11/2018
+ */
+
+
+public class DisplayTime {
+    public static void main(String[] args) throws InterruptedException {
+        DateTime server1 = new DateTime();
+        DateTime server2 = new DateTime();
+        Client client1 = new Client();
```

Comment:
According to hometask you should have different implementations of Clients. That's the main purpose of using Observer pattern. 

Please follow the task and re-write code: 
```
Общий код метода main может выглядеть следующим образом:
public static void main(String[] args) { 
Server server1 = new ServerImplOne(); 
Server server2 = new ServerImplTwo();
Client client1 = new ClientOne(); 
Client client2 = new ClientTwo(); 
Client client3 = new ClientThree(); 
Client client4 = new ClientFour(); 
server1.register(client1); server1.register(client2); server1.register(client3); server2.register(client2); server2.register(client4); for (int i=0;i<10;i++) { server1.process(); server2.process(); //Sleep for 2 seconds      Thread. sleep (2000); } }
```
`Client` is an interface
`Server` is an interface 
`ServerImplOne` is implementation of `Server`,
`ClientOne` is implementation of `Client`
 etc